crates/hotshot-state-prover/src/circuit.rs

```diff
@@ -243,35 +283,71 @@ mod tests {
 
     #[test]
     fn test_circuit_building() {
-        let keys = key_pairs_for_testing();
-        let st = stake_table_for_testing(&keys);
+        let mut prng = test_rng();
+
+        let (bls_keys, schnorr_keys) = key_pairs_for_testing(&mut prng);
+        let st = stake_table_for_testing(&bls_keys, &schnorr_keys);
+
+        let block_comm =
+            VariableLengthRescueCRHF::<F, 1>::evaluate(vec![F::from(1u32), F::from(2u32)]).unwrap()
+                [0];
+        let fee_ledger_comm =
+            VariableLengthRescueCRHF::<F, 1>::evaluate(vec![F::from(3u32), F::from(5u32)]).unwrap()
+                [0];
 
         let lightclient_state = LightClientState {
-            view_number: 0,
-            block_height: 0,
-            block_comm: F::default(),
-            fee_ledger_comm: F::default(),
+            view_number: 24,
+            block_height: 738,
```

Comment:
Should block height usually be smaller than the view number?

Reply:
done in the newest commitment